### PR TITLE
chore(deps): bump github.com/testcontainers/testcontainers-go to v0.28.0

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
## Description
- bump github.com/testcontainers/testcontainers-go to v0.28.0

## Checklist
- [ ] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [ ] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

**Comment by knqyf263** [comment]
The latest aws-sdk-go-v2 added breaking changes?

**Comment by Prajyot-Parab** [comment]
> The latest aws-sdk-go-v2 added breaking changes?

I have lowered `testcontainers/testcontainers-go` to use v0.28.0 for now, hopefully it wont throw any issues. waiting for checks to report status.

File: go.mod [hunk 1]
```diff
@@ -429,5 +437,6 @@ require (
 )
 
 // testcontainers-go has a bug with versions v0.25.0 and v0.26.0
-// ref: https://github.com/testcontainers/testcontainers-go/issues/1782
-replace github.com/testcontainers/testcontainers-go => github.com/testcontainers/testcontainers-go v0.23.0
+// ref: https://github.com/testcontainers/testcontainers-go/issues/1782 (marked as closed)
+// revert the changes if bug still exists
+// replace github.com/testcontainers/testcontainers-go => github.com/testcontainers/testcontainers-go v0.23.0
```
Comment (knqyf263) [inline go.mod:442]: Please remove these lines

Comment (Prajyot-Parab) [reply go.mod]: Fixed.

**Review by knqyf263 — APPROVED** [review]

**Comment by Prajyot-Parab** [comment]
@knqyf263 ptal I have fixed the conflicts.

**Comment by knqyf263** [comment]
@Prajyot-Parab Thanks!

**Pull request merged**